lib/pdf/inspector/text.rb

```diff
@@ -62,6 +64,11 @@ def set_word_spacing(*params)
         @word_spacing << params[0]
       end
       
+      def set_horizontal_text_scaling(*params)
```

Comment:
I've been looking into writing specs for this, and I now understand that the reason why this method, and all other similar methods take the catch all is because it can be a variable argument list, as different PDF operators take different numbers of params. E.g., the `set_text_font_and_size` method (`Tm` PDF operator) takes two arguments.